add_nyt_review_link.py

```diff
@@ -191,8 +195,15 @@ def run(self) -> None:  # overwrites the AbstractBotJob run method
         comment = 'Add NYT review links'
         with open(self.args.file, 'r') as fin:
             review_record_array = json.load(fin)
-            for review_record in tqdm(review_record_array, unit='reviews'):
```

Comment:
I think you may have accidentally reverted changes I made here to simplify the usage of tqdm.

Reply:
yeah, sorry, I did not realize you did the changes after I explained why I use more complex approach with
 manual update https://github.com/RayBB/openlibrary-nyt-bot-temp/pull/6#discussion_r684306216 If you are OK to see the progress bar every other line in logs, I'm OK to put your, simplified version back.